test/e2e/framework/kubernetes/install-retina-helm.go

```diff
@@ -13,8 +13,10 @@ import (
 )
 
 const (
-	createTimeout = 240 * time.Second // windpws is slow
-	deleteTimeout = 60 * time.Second
+	createTimeout  = 240 * time.Second // windpws is slow
+	deleteTimeout  = 60 * time.Second
+	imageRegistry  = "acnpublic.azurecr.io"
+	imageNamespace = "microsoft/retina"
```

Comment:
You are still hardcoding these values, is this necessary? No way to make them parameters?

Reply:
updated the code to set those form pipeline as env.